<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectToStadiaWidget</class>
 <widget class="QWidget" name="ConnectToStadiaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="mainFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="contentFrame">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="titleBarLayout">
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>222</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Connect to a Stadia instance</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="refreshButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Refresh Instance list</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/orbiticons.qrc">
               <normaloff>:/actions/refresh</normaloff>:/actions/refresh</iconset>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>198</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QTableView" name="instancesTableView">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <widget class="orbit_qt::OverlayWidget" name="instancesTableOverlay" native="true">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>100</width>
                  <height>30</height>
                 </rect>
                </property>
                <property name="visible">
                 <bool>false</bool>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
               <item>
                <widget class="QCheckBox" name="rememberCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Connect to the selected instance on Orbits next start</string>
                 </property>
                 <property name="text">
                  <string>Automatically connect to this instance next time</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="connectButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Connect</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>orbit_qt::OverlayWidget</class>
   <extends>QWidget</extends>
   <header>OverlayWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/orbiticons.qrc"/>
 </resources>
 <connections/>
</ui>
